src/Toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ElementType, FC, HTMLAttributes } from 'react';
+import classnames from 'classnames';
+
+export interface ToolbarProps extends HTMLAttributes<HTMLElement> {
+  tag?: ElementType;
+  isMedium?: boolean;
+  isSmall?: boolean;
```

Comment:
Having two boolean flags means that the user could potentially set a conflict state with something like:
```jsx
<Toolbar isMedium isSmall />
```

A nicer API could be designed to not lead to conflicts, with a single parameter like `size?: 'large' | 'medium' | 'small'` (or `'lg' | 'md' | 'sm'` as in other props in reactstrap), where `large`/`lg` would be the default value.

Reply:
Yes, sure. I've named them 'large/medium/small since the original classes in Bootstrap Italia named that way. And here I have a question: is the class 'toolbar-large' needed? There is only 'toolbar-medium' and 'toolbar-small' are added and [large icons are as the default](https://italia.github.io/bootstrap-italia/docs/menu-di-navigazione/toolbar/) (without adding any second class to toolbar).

Reply:
That’s ok to receive “large” or have it as default value. Internally we can filter it as below